lib/harmony/harmonydb/harmonydb.go

```diff
@@ -16,6 +17,7 @@ import (
 	"github.com/jackc/pgx/v5"
 	"github.com/jackc/pgx/v5/pgconn"
 	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/kr/pretty"
```

Comment:
Can we avoid pulling in a random new dependency just to print a network address?